### PR TITLE
Move getting started up

### PR DESCRIPTION
**Pull request by yves-chan**
Currently it looks like this:
image

Didn't know there was a categoryKey setting, right now for VMs it is set to 100.  Will change 'getting started' to have order of 50 so it should be placed above it.

- Getting started
- VM
- etc.

### CODE REVIEW

**Comment by gardnerjr** [comment]
we should probably make some really generic getting started ones that are just markdown + links to resources to put in that category?  otherwise again its a group of one item taking up space?

**Comment by yves-chan** [comment]
@jgardner, Added a general WB that links to some of our docs:
image

File: Workbooks/Azure Monitor - Getting Started/Getting Started Resources/Helpful Links.workbook [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
...ally.md)\r\n\r\n## Samples\r\nBrowse some of our samples [here](https://github.com/microsoft/Application-Insights-Workbooks/tree/master/Documentation/Samples)!\r\n\r\n## Are you coming from View Designer?\r\nHere's how you can make the shift to Workbooks!\r\n- [View Designer to Workbooks Transition Guide](https://github.com/microsoft/Application-Insights-Workbooks/blob/master/Documentation/ViewDesigner/ViewDesignerOverview.md)\r\n- [Accessing Workbooks & Viewing Permissions](https://github.com/microsoft/Application-Insights-Workbooks/blob/master/Documentation/ViewDesigner/AccessPermissions.md)\r\n- [Common Steps](https://github.com/microsoft/Application-Insights-Workbooks/blob/master/Documentation/ViewDesigner/CommonSteps.md)\r\n- [Conversion Options](https://github.com/microsoft/Application-Insights-Workbooks/blob/master/Documentation/ViewDesigner/ConversionOptions.md)\r\n- [Tile Conversions](https://github.com/microsoft/Application-Insights-Workbooks/blob/master/Documentation/ViewDesigner/TileConversions.md)\r\n\r\n---\r\n\r\nFind out more about [contributing](https://github.com/microsoft/Application-Insights-Workbooks/blob/master/Documentation/Contributing.md) to Workbooks!\r\n"
```
Comment (gardnerjr) [inline Workbooks/Azure Monitor - Getting Started/Getting Started Resources/Helpful Links.workbook]: my only nit here would be to use the docs on azure docs if they exist first, falling back to our github ones?  the azure docs are supposed to be our "official" ones, and the github ones kindof the "developer" ones?  this is a good start though!

**Comment by gardnerjr** [comment]
oh and one more *tiny* comment!  i'd move away from using exclamation marks everywhere!  we all know workbooks is exciting!  but not that exciting! :D

**Pull request merged**